scripts/setfit/run_fewshot.py

```diff
@@ -99,6 +101,19 @@ def main():
     for dataset, metric in dataset_to_metric.items():
         few_shot_train_splits, test_data = load_data_splits(dataset, args.sample_sizes, args.add_data_augmentation)
 
+        # Potentially report on naive use of accuracy with an imbalanced test set
+        if metric == "accuracy":
+            counter = Counter(test_data["label"])
+            label_samples = sorted(counter.items(), key=lambda label_samples: label_samples[1])
+            smallest_n_samples = label_samples[0][1]
+            largest_n_samples = label_samples[-1][1]
+            # If the largest class is more than 50% larger than the smallest
+            if largest_n_samples > smallest_n_samples * 1.5:
+                warnings.warn(
+                    f"The test set has a class imbalance ({', '.join(f'label {label} w. {n_samples} samples' for label, n_samples in label_samples)})"
+                    ", but is evaluated using `accuracy`, which may lead to an evaluation that does not correspond with true model performance.",
```

Comment:
Same as above, maybe we can print out no matter what metric the dataset is using?
Like:
```suggestion
                    f", but is evaluated using `{metric}`, which may lead to an evaluation that does not correspond with true model performance.",
```

Reply:
I see, I think those comments are good ideas. It would be more like a `logging.info` call than a `warnings.warn`, as it would just be informative at all times.

Reply:
Maybe it could be a warning since it pointing out there is an unbalanced dataset?
Just sharing thoughts :)

Reply:
I love the thoughts! My bad, I thought you proposed to *always* print the test set distribution, even when balanced.

Reply:
Sorry, forgive my limited language skills 😂